packages/Input/__tests__/Input.test.tsx

```diff
@@ -1,47 +1,131 @@
-import { mount, render, shallow } from 'enzyme';
-import expect from 'expect';
-import React from 'react';
-import Input from '../src';
-
-const testProps = {
-  onChange: () => { return; },
-  placeholder: 'Placeholder text'
-};
-
-describe('Input component', () => {
-  test('should be defined and renders correctly (snapshot)', () => {
-    const wrapper = shallow(<Input hideTypeToggle type="password" {...testProps} />);
+import { fireEvent, render } from "@testing-library/react";
+import { mount } from "enzyme";
+import "jest-dom/extend-expect";
+import React from "react";
+import Input from "../src";
+
+const defaultProps = (override: object = {}) => ({
+  onChange: () => void 0,
```

Comment:
Why are you using the `void`? it's meant to prevent side effects but can't see any... in any case should it's better practice to use `e.preventDefault`.